tests/app-test.js

```diff
@@ -0,0 +1,17 @@
+const assert = require('chai').assert
+const roman = require('../app')
+
+describe('convert arabic numerals to roman numerals', function() {
+  it('accurately converts to roman numerals', function() {
+    assert.deepEqual(roman.convert(5), 'V')
```

Comment:
`deepEqual` is typically used for objects or arrays. `equal` or `strictEqual` would have sufficed. I like the variety of test cases though 👍